tests/unit/createBirthdaySquad.test.ts

```diff
@@ -0,0 +1,254 @@
+import "@/testUtils/unit/mockDb";
+import "@/testUtils/unit/mockEventBridge";
+
+import {
+  EventBridgeClient,
+  PutEventsCommand,
+} from "@aws-sdk/client-eventbridge";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import type { Mock } from "vitest";
+import { afterEach, beforeEach, describe, expect, it, vi } from "vitest";
+
+import * as getSquadMembers from "@/db/queries/getSquadMembers";
+import { squadJoins, users } from "@/db/schema";
+import type { Events } from "@/events";
+import { handler } from "@/functions/events/createBirthdaySquad";
+import { BIRTHDAY_SQUAD_SIZE } from "@/functions/utils/constants";
+import { openConversation } from "@/services/slack/openConversation";
+import { seedSquadJoins } from "@/testUtils/seedSquadJoins";
+import { testDb } from "@/testUtils/testDb";
+import { mockEventBridgePayload } from "@/testUtils/unit/mockEventBridgePayload";
+import { sendMockSqsMessage } from "@/testUtils/unit/sendMockSqsMessage";
+
+dayjs.extend(utc);
+
+const constants = vi.hoisted(() => ({
+  birthdayPerson: "U001",
+  eventId: "E001",
+  teamId: "T001",
+  otherUserIds: ["U002", "U003", "U004", "U005", "U006"],
+  converastionId: "CH001",
+}));
+
+vi.mock("@/services/slack/openConversation", async () => ({
+  openConversation: vi.fn().mockResolvedValue(constants.converastionId),
+}));
+
+describe("CreateBirthdaySquad", () => {
+  let openConversationMock: Mock;
```

Comment:
Do we need this mock? As far as I see we are always returning the same conversationId so the mock above is enough.